client/src/components/Footer/FooterBottom.jsx

```diff
@@ -10,7 +10,18 @@ function FooterBottom() {
           Github Project Link - August 2024
         </a>
       </span>
+      <div className="relative w-full flex flex-col items-center content-center sm:flex-row sm:items-start justify-center mt-12 rounded-3xl bg-[#9BC25B]">
+        <div className="px-8 py-8 flex content-center">
+      Disclaimer: 
+      This website and its associated services are provided for demonstrative and educational purposes only. The content and features, including the ability to report bee swarms, are part of a student project and may not be fully functional or accurate. The information presented about bees is intended solely for informational purposes and should not be relied upon for professional advice or real-world applications.
```

Comment:
My suggestion is to change the word student to the word demonstrative

Reply:
I changed it to demonstrative. I also changed the user data in the footer to add Shimin(Natalie) and me to the site since I was in the footer section. Should I have created a new branch/pull request for that?